doc/user/content/integrations/websocket-api.md

```diff
@@ -52,13 +52,18 @@ The WebSocket API provides two modes with slightly different transactional seman
 ### Endpoint
 
 ```
-https://<MZ host address>/api/experimental/sql
+wss://<MZ host address>/api/experimental/sql
 ```
 
-Accessing the endpoint requires [basic authentication](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#basic_authentication_scheme). Reuse the same credentials as with `psql`:
+To authenticate, send an initial text or binary message containing a JSON object with keys:
+
+- `user_id`: Your email to access Materialize.
```

Comment:
This should probably be `user`, to match what `psql` calls this? "User ID" would be a new term, I think.

Reply:
Oh, I see, "user ID" is what HTTP basic authentication calls this parameter. Sigh. We'll be inconsistent with something either way. Whatever you think.